dotcom-rendering/src/web/components/ImageComponent.tsx

```diff
@@ -282,6 +306,7 @@ export const ImageComponent = ({
 	) {
 		return (
 			<div
+				id={element.position ? `img-${element.position}` : ''}
```

Comment:
Used for hash navigation

Reply:
This will need to be `!!element.position` as per our new ESLint rules

Reply:
I was reviewing some outdated code. Move along!